application/liveness/liveness.go

```diff
@@ -0,0 +1,174 @@
+package liveness
+
+import (
+	"fmt"
+	"net"
+	"strconv"
+	"time"
+	"encoding/csv"
+	"os"
+	"os/exec"
+)
+
+type LivenessTester interface {
+    PhantomIsLive(addr string, port uint16) (bool, error)
+}
+
+type Cache_element struct {
+    is_live 	bool

```

Comment:
swap for mixedCase (camel case names) - [ref](https://betterprogramming.pub/naming-conventions-in-go-short-but-descriptive-1fa7c6d2f32a)